azext_edge/edge/params.py

```diff
@@ -539,6 +539,14 @@ def load_iotops_arguments(self, _):
                 "used, a system provided self-signed trust bundle is configured.",
                 arg_group="Trust",
             )
+            context.argument(
+                "user_trust",
+                options_list=["--user-trust", "--ut"],
+                arg_type=get_three_state_flag(),
+                help="Skip the deployment of the system CertManager and TrustManager "
```

Comment:
after review of official docs, let's use this verbiage `cert-manager` and `trust-manager`